src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -83,6 +83,55 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
   /** Inits registered tokens at startup */
   initRegisteredTokensFromCsv()
 
+  /** Inits governance processing */
+  system.scheduler.scheduleOnce(5.seconds)(GovernanceProcessor.processGovernance())
```

Comment:
Since we are storing governance data per-block, can the new table not be populated as part of the regular Lorre iteration process?

Reply:
ok, changed